config-controller/config/rbac/role.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role
+rules:
+- apiGroups:
+  - config.stackrox.io
+  resources:
+  - policies
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - config.stackrox.io
+  resources:
+  - policies/finalizers
```

Comment:
finalizers?

Reply:
This was part of the scaffolding from operator-sdk.  I assume it is to allow the operator/controller to add or remove finalizers from policies.  I believe this is something that we would have to choose to add as needed; we don't add a finalizer by default.